client/src/components/ChatSideBanner/ChatSideBanner.tsx

```diff
@@ -7,6 +7,7 @@ import { User } from '../../interface/User';
 import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
 import Search from '../Search/Search';
 import AuthMenu from '../AuthMenu/AuthMenu';
+import Scoller from '../Scoll/Scrollcomponent';
```

Comment:
```suggestion
import Scroller from '../Scoll/Scrollcomponent';
```
update below as well